Setup_py.bat

```diff
@@ -0,0 +1,14 @@
+mkdir build_py
```

Comment:
We need to remove these scripts. They aren't standardized yet and should be discussed how we want to do this.